omniwitness/configs.go

```diff
@@ -76,11 +60,12 @@ func (config LogConfig) AsLogMap() (map[string]witness.LogInfo, error) {
 	return logMap, nil
 }
 
-// LogInfo contains the configuration options for a log: its identifier, hashing
-// strategy, and public key.
+// LogInfo contains the details about a log.
 type LogInfo struct {
 	Origin        string `yaml:"Origin"`
 	PublicKey     string `yaml:"PublicKey"`
 	PublicKeyType string `yaml:"PublicKeyType"`
+	Url           string `yaml:"URL"`
```

Comment:
nit: this should be URL (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
Fixed.